exercises/word-count/example.go

```diff
@@ -5,20 +5,24 @@ import (
 	"strings"
 )
 
-const testVersion = 2
+const testVersion = 3
 
+// Frequency is a map of the frequency of occurrence keyed to the unique word.
 type Frequency map[string]int
 
+// WordCount returns the map of frequency of words based on the input phrase.
 func WordCount(phrase string) Frequency {
 	freq := Frequency{}
 	for _, word := range strings.Fields(normalize(phrase)) {
+		word = strings.Trim(word, "'")
 		freq[word]++
 	}
 	return freq
 }
 
 func normalize(phrase string) string {
-	r, _ := regexp.Compile(`[^\w]`)
+	//  Allow for apostrophes in words.
```

Comment:
I thought the same but it's sometimes hard to understand the regex content, but am OK to remove it if you think it's redundant.